src/RangeSlider.elm

```diff
@@ -24,7 +25,25 @@ type alias CommonAttributes =
 
 snapValue : Float -> Float -> Float
 snapValue value step =
-    toFloat (round (value / step)) * step
+    let
+        stepDecimals =
+            step
```

Comment:
Trying to understand what this is doing ... is it extracting the number of decimals points of the step value to set the precision?

Reply:
Doesnt really seem to be a hugely neat way to do this really 😅just had a quick look and its mainly string manipulation